systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaTopicResource.java

```diff
@@ -55,4 +57,18 @@ public static void replaceTopicResource(String resourceName, Consumer<KafkaTopic
     public static void replaceTopicResourceInSpecificNamespace(String resourceName, Consumer<KafkaTopic> editor, String namespaceName) {
         ResourceManager.replaceCrdResource(KafkaTopic.class, KafkaTopicList.class, resourceName, editor, namespaceName);
     }
+
+    public static KafkaTopicList getAllKafkaTopicsWithPrefix(String namespace, String prefix) {
```

Comment:
Is this really something we expect to have a general use? If not, should it be instead in the test? I know you didn't started this and you just continue in it ... but this idea that everything is added to the resource classes is a bit weird. It also brings the different resource classes out of sync as each now has different methods etc. 

My rule of thumb would be that if it doesn't seem to be good enough to add to the `ResourceType` interface and implement it in all resource classes, maybe it is not worth having it here?

Reply:
I think that should be answered byt @im-konge @Frawless or @see-quick as they know better this.
I can put it into `io.strimzi.systemtest.utils.kafkaUtils.KafkaTopicUtils` if it's better location.

Reply:
Move it to `KafkaTopicUtils`. In the resource classes we should keep methods relevant to topic creation, deletion or replace .

Reply:
Already did. ;)